File: Problema3/Fecha.java
class Fecha {

    private int mes;
    private int dia;
    private int anio;

    public Fecha(int mes, int dia, int anio) {
        this.mes = mes;
        this.dia = dia;
        this.anio = anio;
    }

    public void mostrarFecha() {
        System.out.println(mes + "/" + dia + "/" + anio);
    }
}

public class Main {

    public static void main(String[] args) {

        Fecha fecha1 = new Fecha(8, 30, 2026);

        System.out.print("Fecha: ");
        fecha1.mostrarFecha();
    }
}

File: Problema3/fecha.py
class Fecha:

    def __init__(self, mes, dia, anio):
        self.mes = mes
        self.dia = dia
        self.anio = anio

    def mostrar_fecha(self):
        print(f"{self.mes}/{self.dia}/{self.anio}")


fecha1 = Fecha(8, 30, 2026)

print("Fecha: ", end="")
fecha1.mostrar_fecha()
